docs/admin.md

```diff
@@ -426,6 +426,35 @@ Content-Type: application/zip
 Content-Disposition: attachment; filename="alice.cozy.localhost - part001.zip"
 ```
 
+## OAuth clients usage
+
+### GET /instances/:domain/clients-usage
+
+This endpoint returns the number of user-connected OAuth clients, the limit set
+on the Cozy and if this limit has been reached or even exceeded.
+If there is no limit, the attribute won't be present in the response.
```

Comment:
You are right

Reply:
Now I understand why the disk usage int attributes are marshalled into strings. I'll do the same for the clients usage limit and count.

cc @Ldoppea this might change how you use the data in the flagship app

Reply:
They are marshaled into strings to avoid losing precision for big numbers. But I think you are looking at a pointer to a number for `limit` type. It allows to `omitempty` the `nil` value, and serializes it to a number even if it is 0.

Reply:
Ah perfect! I'll go that route.